src/odemis/acq/stream/_static.py

```diff
@@ -216,6 +216,11 @@ def __init__(self, name, raw, *args, **kwargs):
             self.zIndex = model.IntContinuous(0, (0, raw[0].shape[0] - 1))
             self.zIndex.subscribe(self._on_zIndex)
 
+            # define a projection
+            # TODO: add support for multiple types of projections (max, min, avg, sum, ...)
```

Comment:
"projection" is very generic. Here, this refers to showing multiple Z levels simultaneously. 
=> I would change the comment to:
```
Option to see all Z indices at the same time using maximum intensity projection (MIP)".
TODO: support other methods for multi-focus fusion?
```